btcpay-dropbox/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7.2-alpine3.9
+
+WORKDIR /btcpay-dropbox
+
+COPY . /btcpay-dropbox
+
+RUN pip install dropbox
+
```

Comment:
Mssing `ENV DROPBOX_TOKEN` so `DROPBOX_TOKEN` get implicitely passed from the host to the container.